core/web_password_reset.js

```diff
@@ -143,7 +143,7 @@ class WebPasswordReset {
                     }
 
                     const message = {
-                        to: `${user.properties[UserProps.RealName] || user.username} <${
+                        to: `${user.realName()} <${
```

Comment:
See above - also, maybe a utility method to format an email address would be handy?